sdk/tests/deqp/functional/gles3/es3fFboInvalidateTests.js

```diff
@@ -859,6 +860,9 @@ es3fFboInvalidateTests.InvalidateSubFboUnbindReadCase.prototype.render = functio
     ctx.disable(gl.DEPTH_TEST);
     ctx.disable(gl.STENCIL_TEST);
 
+    ctx.clearColor(0.25, 0.5, 0.75, 1);
+    ctx.clear(gl.COLOR_BUTTER_BIT | gl.DEPTH_BUFFER_BIT | gl.STENCIL_BUFFER_BIT);
```

Comment:
C++ code:
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/functional/es3fFboInvalidateTests.cpp#1002
Align with C++ code to make debugging easier.